docs/shop4cfcontext.jsonld

```diff
@@ -0,0 +1,29 @@
+{
+  "@context": {
+    
+    "isDefinedBy": {
+      "@id": "https://raw.githubusercontent.com/matteopantano/data-models/master/docs/shop4cfdefinitions.jsonld#definitions/isDefinedBy",
+      "@type": "ngsi-ld:Relationship"
+    },
+    
+    "involves": {
+      "@id": "https://raw.githubusercontent.com/matteopantano/data-models/master/docs/shop4cfdefinitions.jsonld#definitions/involves",
+      "@type": "ngsi-ld:Property"
+    },
+    
+    "happensAt": {
+      "@id": "https://raw.githubusercontent.com/matteopantano/data-models/master/docs/shop4cfdefinitions.jsonld#definitions/happensAt",
+      "@type": "ngsi-ld:Property"
+    },
+    
+    "workParameters": {
+      "@id": "https://raw.githubusercontent.com/matteopantano/data-models/master/docs/shop4cfdefinitions.jsonld#definitions/workParameters",
+      "@type": "TO BE DEFINED"
```

Comment:
Why not `ngsi-ld:Property`?

Reply:
I was not sure about this, we said that work parameters will need to be defined by every pilot and it might be that the content of them varies and Property is not enough. Am I wrong? What do you think?

Ideally every pilot could create its own context and inside it specify its work parameters, this would be pretty cool. However, do you think it is feasible?

Reply:
I would say the `workParameters` value should always be a dictionary, thus a Property. Like in our example:

```
    "workParameters": {
        "type": "Property",
        "value": {
            "materialAmount": 8
        }
    },
```

But specific fields in this dictionary (e.g. `materialAmount`) are up to specific use cases. 

But no matter what specific fields are used, `workParameters` still remains a Property. At least this is my understanding.

Reply:
Ok for property